libs/dh/admin/data-access-api/src/lib/dh-admin-user-management-data-access-api.store.ts

```diff
@@ -184,6 +210,14 @@ export class DhAdminUserManagementDataAccessApiStore
     this.patchState({ statusFilter: userStatus });
   }
 
+  updateActorFilter(actorId: string | undefined) {
+    this.patchState({ actorIdFilter: actorId });
+  }
+
+  updateUserRoleFilter(userRole: string[]) {
+    this.patchState({ userRoleFilter: userRole || [] });
```

Comment:
|| [] should not be necessary

Reply:
it is necessary, but moved to filter component